src/consumption/elevate.js

```
@@ -0,0 +1,1827 @@
+			await uploadFile(dirPath, fileName, fileUploadUrl)
+			// delete folder after upload
+			await deleteFolderRecursive(path.join(mainPath, uniqueId))
+
+			resolve({
+				message: 'Template edited successfully',
+				filePath: uploadedFilePath,
+			})
+		} catch (error) {
+			reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)